pom.client.xml

```diff
@@ -38,7 +38,4 @@
     </dependencies>
   </dependencyManagement>
 
-  <modules>
```

Comment:
Use inheritance over multi-module for a simpler module management.

As the number of child SDKs grow, each of them is independent, with the only commonality being the parent pom. The parent pom isn't aware of the children either so child projects can change without modifying this section.

This file serves as the universal parent for dependencies, plugins, etc. The only time to modify this file is when we decide to upgrade a dependency version or alter a build process.